File: java/src/main/java/toolgood/flowVision/Flows/FactoryProcedureItemWork.java
package toolgood.flowVision.Flows;

import com.alibaba.fastjson.JSONObject;

public class FactoryProcedureItemWork {
    public String CategoryCode;
    public String Category;
    public String Code;
    public String Name;

    final static FactoryProcedureItemWork parse(JSONObject jsonObject) {
        FactoryProcedureItemWork result = new FactoryProcedureItemWork();
        result.CategoryCode = jsonObject.getString("categoryCode");
        result.Category = jsonObject.getString("category");
        result.Code = jsonObject.getString("code");
        result.Name = jsonObject.getString("name");

        return result;
    }
}

File: csharp/ToolGood.FlowVision/Flows/FactoryProcedureItemWork.cs
using System.Text.Json.Serialization;

namespace ToolGood.FlowVision.Flows
{
	public sealed class FactoryProcedureItemWork
	{
		[System.Text.Json.Serialization.JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
		public string CategoryCode { get; set; }

		[System.Text.Json.Serialization.JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
		public string Category { get; set; }

		[System.Text.Json.Serialization.JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
		public string Code { get; set; }

		[System.Text.Json.Serialization.JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
		public string Name { get; set; }
	}
}